src/layer/GeoJSON.js

```diff
@@ -232,7 +232,10 @@ function _pointToLayer(pointToLayerFn, geojson, latlng, options) {
 // Creates a `LatLng` object from an array of 2 numbers (longitude, latitude)
 // or 3 numbers (longitude, latitude, altitude) used in GeoJSON for points.
 export function coordsToLatLng(coords) {
-	return new LatLng(coords[1], coords[0], coords[2]);
+	if (coords.length === 3) {
+		return new LatLng(coords[1], coords[0], coords[2]);
+	}
+	return new LatLng(coords[1], coords[0]);

```

Comment:
If `coordsToLatLng` is overwritten like in many SO answers suggested, the check `coords.length === 3` will not be done.

For example a simple switch of the array if source is already latlng:
```
coordsToLatLng(coords) {
	return new LatLng(coords[0], coords[1], coords[2]);
```